tests/testthat/test-read_vic_schools_fte.R

```diff
@@ -1,3 +1,98 @@
-test_that("multiplication works", {
-  expect_equal(2 * 2, 4)
+# testing read_vic_schools_fte function
+# there is no URL, urls$read_vic_schools_fte is NULL
+
+test_that("read_vic_schools_fte works", {
+
+  fte <- read_vic_schools_fte()
+
+  expect_s3_class(fte, 'data.frame')
+  expect_equal(class(fte$school_no), 'numeric')
+  expect_equal(class(fte$year), 'character')    # should that be 'Date'?
+
+  expect_equal(colnames(fte), c("education_sector",
```

Comment:
can you add an issue for this one to return a long format dataframe, and some filter options to just return primary/highschool, male/female, totals

Reply:
Ok, will do.

Reply:
https://github.com/djpr-data/djprdata/issues/28